glide.lock

```diff
@@ -1,11 +1,21 @@
-hash: 5d0a7018b7275c53493a2cda398bd29693ea8097abd3d8e69d6f16889adfff23
-updated: 2017-06-30T13:36:26.038479713Z
+hash: 956185a890fa281a7b6bb13c215588e9eadf13e11efaa449b25e416914d8899c
+updated: 2017-09-13T07:13:18.994310187Z
 imports:
```

Comment:
Did we upgrade all deps?  We should.

Reply:
Marco, a "glide update" results in an empty diff, which makes me believe that all the dependencies are up-to-date.